arcgis-ios-sdk-samples/Content Display Logic/Controllers/ContentTableViewController.swift

```diff
@@ -187,34 +200,37 @@ class ContentTableViewController: UITableViewController {
         let controller = storyboard.instantiateInitialViewController()!
         controller.title = sample.name
         
-        // must use the presenting controller when opening from search results or else splitViewController will be nil
+        // Must use the presenting controller when opening from search results or else splitViewController will be nil.
         let presentingController: UIViewController? = searchEngine != nil ? presentingViewController : self
             
         let navController = UINavigationController(rootViewController: controller)
         
-        // don't use large titles on samples
+        // Don't use large titles on samples.
         controller.navigationItem.largeTitleDisplayMode = .never
         
-        // add the button on the left on the detail view controller
+        // Add the button on the left on the detail view controller.
         controller.navigationItem.leftBarButtonItem = presentingController?.splitViewController?.displayModeButtonItem
         controller.navigationItem.leftItemsSupplementBackButton = true
         
-        // present the sample view controller
+        // Present the sample view controller.
         presentingController?.showDetailViewController(navController, sender: self)
         
-        // create and setup the info button
+        // Create and setup the info button.
         let infoBBI = SourceCodeBarButtonItem()
         infoBBI.readmeURL = sample.readmeURL
         infoBBI.navController = navController
-        controller.navigationItem.rightBarButtonItem = infoBBI
+        infoBBI.sample = sample
```

Comment:
This line is wrong, I assume it should be `favoritesBarButtonItem.sample = sample` ?

Also let's rename `infoBBI` to `sourceCodeBarButtonItem`.